designs/administrator/olm/operand-details-grouping/index.md

```diff
@@ -0,0 +1,36 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Improve grouping and editing on Operand Details view
+
+When creating an operand, fairly complex nesting and hierarchy of fields is used to set values on the operand. Once created however, the Details tab for that operand does not show this hierarchy and all fields are shown at the same flat level, with somewhat of a mix of spec and status descriptors, with some hierarchical section headers missing as well. This can make this view challenging to interpret.
+
+This design describes surfacing the common spec descriptors first, the status descriptors next, then the rest of the spec descriptors, with a divider separating them. Groups of spec descriptors will now have their grouping reflected using a box for the first level, with level indicator labels conveying any further hierarchy. An option to edit these groups using their existing form view will also be available.
+
+## Separating Spec and Status descriptors
+
+![Dividing groups](img/8-1-divider.png)
+- A new divider will be added between the status and spec descriptor sections to visually separate them.
+
+## Grouping Spec descriptors
+
+![Conveying groups](img/9-1-group.png)
+- The top level of spec descriptors' hierarchy will now be shown grouped in a box.
+- Each field has a level indicator to exactly convey its hierarchy, which would be important for fields beyond the first depth level.
+
+![Conveying groups popover](img/9-2-groupPop.png)
+- Each descriptor will support an existing popover style that shows more information on the header click, including the description and full hierarchy level path, useful if the level indicator is truncated. 
+
+## Allowing quick access to edit form from a Spec group
+
+![Group editing](img/7-1-editGroup.png)
```

Comment:
I can't help but feel like the descriptor path adds a lot of visual noise to the page. Maybe that's totally acceptable because it is important information for the user to see right here at a glance on this page. But I also wonder if we could fold some questions around this into an upcoming usability test to see if having this information in the popover would be sufficient. Ignore me if you've already gotten this feedback from users :)

Reply:
Thanks a lot for the feedback, Liz. After talking it over with Tony and Jon, we agree and are going to forego the path on the details page for now, since the popover contains the same info and wasn't there when this was initially designed.

We figure we can try this page out without it, and then see if we really need to add it.